addons/web/static/src/js/views/kanban/kanban_column_quick_create.js

```diff
@@ -32,6 +32,7 @@ var ColumnQuickCreate = Widget.extend({
      */
     init: function (parent, options) {
         this._super.apply(this, arguments);
+        this.applyExampleText = options.apply_example_text;
```

Comment:
Combo comments:
- Put the default value here:
 `this.applyExampleText = options.applyExampleText || _t("Use This For My Kanban");`
- Rename to applyExample**s**Text for consistency
- I think you can use camelCase instead of snake_case everywhere for this new property?
  The only time where we use snake_case in JS is when the property comes from the xml or from a python controller